scripts/install-mender.sh

```diff
@@ -328,6 +334,23 @@ EOF
             echo "  Success!"
         fi
     fi
+
+    # Setup for mender-gateway
+    if [[ "$SELECTED_COMPONENTS" == *"mender-gateway"* ]]; then
+        if [ "$DEMO" -eq 1 ]; then
+            echo "  Setting up mender-gateway with demo configuration, certificates and key"
+            url="$MENDER_COMMERCIAL_DOWNLOAD_URL$(printf ${MENDER_GATEWAY_EXAMPLES_URL_PATH_F} $VERSION $VERSION)"
+            curl -fLsS -H "Authorization: Bearer $JWT_TOKEN" -O "$url" ||
+                    (echo ERROR: Cannot get mender-gateway-examples from $url; exit 1)
```

Comment:
* it will never contain URL encoded characters? (see the wget excursion we have enjoyed so much earlier)
* the exit here will only exit the subshell, is that on purpose?

Reply:
I moved it into an explicit `if` function

Reply:
the if looks good. another option was to use the group command: `... || {...; exit 1; }` but it is fine.
the wget/curl fun was due to curl not de-encoding the URL-encoded filenames, but if you have it under control then it is ok.